pyfar/dsp/dsp.py

```diff
@@ -1482,20 +1507,27 @@ def deconvolve(system_output, system_input, fft_length=None, freq_range=None,
            sweeps. Directors cut." J. Audio Eng. Soc. 49(6):443-471,
            (2001, June).
     """
-
     # Check if system_output and system_input are both type Signal
     if not isinstance(system_output, pyfar.Signal):
         raise TypeError('system_output has to be of type pyfar.Signal')
     if not isinstance(system_input, pyfar.Signal):
         raise TypeError('system_input has to be of type pyfar.Signal')
 
+    # Check frequency range parameter
+    if frequency_range is None and freq_range is None:
+        frequency_range = (0, system_input.sampling_rate/2)
```

Comment:
```suggestion
```
this is also changing to logic of this function, it is allowed not to set the frequency_range parameter.